Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -25,9 +25,13 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
 {
     batteryStateOfChargeService_.addData(batteryData);
 
+
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
-
-    // TODO: Print out time until the battery is fully charged or depleted.
-}
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed();
+           if(batteryStateOfChargeService_.isCharging()){
```

Comment:
Align this if statement with `QTextStream(stdout) above.`
Also, the style on the team is to have opening braces on the following line:
`````
if(blah)
{
   //code
}